src/EditView.vala

```diff
@@ -298,6 +301,14 @@ class EditView: Gtk.DrawingArea, Gtk.Scrollable {
 		});
 	}
 
+	public void paste_primary() {
```

Comment:
maybe this method should be private?

Reply:
It seems so. I'll be honest: as it's my first time with Vala I've tried to make the least changes possible, so I just copy-pasted `paste` and edited the constant.